packages/lesswrong/lib/utils/asyncUtils.ts

```diff
@@ -29,3 +29,16 @@ export const asyncForeachSequential = async <T>(list: Array<T>, fn: (x:T,i:numbe
 export const asyncForeachParallel = async <T>(list: Array<T>, fn: (x:T, i:number)=>Promise<void>): Promise<void> => {
   await Promise.all(list.map((x,i) => fn(x,i)));
 }
+
+export const promisify = (fn: Function) =>
```

Comment:
?!  Any reason we couldn't use the native `utils.promisify` instead?  Seems like it's doing the same thing, except with less type-safety

Reply:
`utils.promisfy` is a nodejs function not available on the client, and this is used in shared code. Could use a better type annotation.